builtin/game/statbars.lua

```diff
@@ -1,39 +1,39 @@
 -- cache setting
 local enable_damage = core.settings:get_bool("enable_damage")
 
-local health_bar_definition = {
-	hud_elem_type = "statbar",
-	position = {x = 0.5, y = 1},
-	text = "heart.png",
-	text2 = "heart_gone.png",
-	number = core.PLAYER_MAX_HP_DEFAULT,
-	item = core.PLAYER_MAX_HP_DEFAULT,
-	direction = 0,
-	size = {x = 24, y = 24},
-	offset = {x = (-10 * 24) - 25, y = -(48 + 24 + 16)},
-}
-
-local breath_bar_definition = {
-	hud_elem_type = "statbar",
-	position = {x = 0.5, y = 1},
-	text = "bubble.png",
-	text2 = "bubble_gone.png",
-	number = core.PLAYER_MAX_BREATH_DEFAULT,
-	item = core.PLAYER_MAX_BREATH_DEFAULT * 2,
-	direction = 0,
-	size = {x = 24, y = 24},
-	offset = {x = 25, y= -(48 + 24 + 16)},
+local bar_definitions = {
+	hp = {
+		hud_elem_type = "statbar",
+		position = {x = 0.5, y = 1},
+		text = "heart.png",
+		text2 = "heart_gone.png",
+		number = core.PLAYER_MAX_HP_DEFAULT,
+		item = core.PLAYER_MAX_HP_DEFAULT,
+		direction = 0,
+		size = {x = 24, y = 24},
+		offset = {x = (-10 * 24) - 25, y = -(48 + 24 + 16)},
+	},
+	breath = {
+		hud_elem_type = "statbar",
+		position = {x = 0.5, y = 1},
+		text = "bubble.png",
+		text2 = "bubble_gone.png",
+		number = core.PLAYER_MAX_BREATH_DEFAULT * 2,
+		item = core.PLAYER_MAX_BREATH_DEFAULT * 2,
```

Comment:
This is a regression, with the change in this pr I get double bubble icons that disappear in pairs instead of single icons that disappear one at a time. Strangely, the bubble backgrounds also only exist beneath half the icons... presumably the `item` field is halved at some point:
![Screenshot-2022-01-28-18:45:23](https://user-images.githubusercontent.com/2219707/151636588-4a39458e-f512-437e-ab9a-590f12df8e93.png)

Reply:
Indeed, it's right here:
![grafik](https://user-images.githubusercontent.com/1042418/151639682-38d53431-2404-4f80-8340-903dcebd7427.png)

Reply:
Indeed, I was too sloppy there - I noted that clearly the initial def was wrong, as fg & bg use the same units (half elements), so both had to be multiplied by two. I missed the multiplier sfan has shown though.